corehq/motech/repeaters/migrations/0015_drop_receiverwrapper_couchdb.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.25 on 2024-04-02 14:38
+
+from django.conf import settings
+from django.db import migrations
+
+from corehq.util.django_migrations import skip_on_fresh_install
+from couchdbkit.client import Database, ResourceNotFound
+
+
+@skip_on_fresh_install
+def _delete_receiverwrapper_couchdb():
```

Comment:
I think this needs to accept two params, `apps, schema_editor`. e.g.
https://github.com/dimagi/commcare-hq/blob/686971041bcfa6663945aaa8659d39fc860498b7/corehq/project_limits/migrations/0002_ratelimitedtwofactorlog.py#L7